pom.xml

```diff
@@ -129,6 +129,7 @@
     <module>runners</module>
     <module>examples/java</module>
     <module>sdks/java/maven-archetypes</module>
+    <module>io</module>
```

Comment:
This will likely need a rebase / manual resolve (since #90 is in process of merging).